.github/workflows/tests.yml

```diff
@@ -14,6 +14,10 @@ jobs:
     name: 'Typecheck'
     runs-on: macos-latest
     steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Hmm it can technically be done with concurrency but it's more difficult because this workflow runs on pull_requests and push. IMO i'd rather merge this so we have the functionality and if you want to investigate into concurrency we can improve on it that way 👍🏼

Reply:
Since this is not tied to the PR I'd prefer to split it out, so we can properly discuss it before.

Reply:
hm okay then